tools/rosgraph/src/rosgraph/xmlrpc.py

```diff
@@ -76,9 +76,36 @@ def isstring(s):
     except NameError:
         return isinstance(s, str)
 
-class SilenceableXMLRPCRequestHandler(SimpleXMLRPCRequestHandler):
 
-    protocol_version = 'HTTP/1.1'
+def check_linux_kernel(major, minor):
+    """Small helper version to check that the current platform kernel is a
+    newer or the same as the input
+    """
+    if platform.system()!='Linux':
```

Comment:
nitpick

```suggestion
    if platform.system() != 'Linux':
```

Reply:
I moved the check outside the function since it didn't seem right to return true if not Linux.
Also I forgot to import platform so hopefully the tests pass this time.